### ISSUE TITLE
Add feature to show checkboxes or tasklists as in GitHub

### ISSUE THREAD

**Issue by dtchanpura** [fabiocolacio/Marker#218]
GitHub markdown supports creation of checkboxes by writing `[ ]` (space inside square brackets) and selected checkbox by `[x]` (x inside square brackets)

Just like as follows
- [ ] not selected
- [x] selected

Reference: https://blog.github.com/2014-04-28-task-lists-in-all-markdown-documents/

**Comment by fabiocolacio** [comment]
I agree that this is a nice feature to add. We use [scidown](https://github.com/Mandarancio/Scidown), as a markdown parser, so you might consider opening this issue there also, but I may implement this myself if I can find the time.

**Comment by smithfred** [comment]
Would be better to support the [full set](https://github.github.com/gfm/) of GFM extensions if possible, since it's a widely used set of extensions (and is fully specified against CommonMark).

**Comment by stefs** [comment]
Here is the relevant scidown issue: https://github.com/Mandarancio/scidown/issues/7
However scidown seems to be unmaintained/abandoned. The last released version is 4 years old.
Maybe it is better to switch to a different parser because of this.
